### PR TITLE
TASK: Update flow version require for neos/welcome

### LINKED ISSUE #46: Update neos/welcome flow dependency version on branching

### PR DESCRIPTION
**Pull request by albe**
Resolves #46

### CODE REVIEW

**Comment by kdambekalns** [comment]
Let's have a quick discussion about this on the issue…

File: Build/set-dependencies.sh [hunk 1]
```diff
@@ -115,5 +115,6 @@ for PACKAGE in Neos.Eel Neos.FluidAdaptor Neos.Kickstarter ; do
 done
 
 php "${COMPOSER_PHAR}" --working-dir=Packages/Framework/Neos.Kickstarter require --no-update "neos/fluid-adaptor:~${BRANCH}.0"
+php "${COMPOSER_PHAR}" --working-dir=Packages/Application/Neos.Welcome require --no-update "neos/flow:~${BRANCH}.0"
```
Comment (kdambekalns) [inline Build/set-dependencies.sh:118]: In any case this needs to be amended with a push in the package, otherwise that change will never make it upstream…

**Review by kdambekalns — CHANGES_REQUESTED** [review]

File: Build/set-dependencies.sh (hunk 1 above)
Comment (albe) [reply Build/set-dependencies.sh:118]: Doesn't the create-branch script do that after invoking set-dependencies? https://github.com/neos/flow-development-distribution/blob/master/Build/create-branch.sh#L50

Comment (kdambekalns) [reply Build/set-dependencies.sh:118]: 🤔 TBH I just checked the repo and… https://github.com/neos/flow-welcome/tree/7.0 is not where it should be, then.

**Comment by albe** [comment]
So, should we merge this for now given it *should* work as of the current create-branch.sh?

File: Build/set-dependencies.sh (hunk 1 above)
Comment (kdambekalns) [reply Build/set-dependencies.sh:118]: Indeed, Welcome is pushed, my bad!

**Review by kdambekalns — APPROVED** [review]

**Pull request merged**